docker-compose.yml

```diff
@@ -27,9 +26,34 @@ services:
       # existing content of the image at the same location
       - /app/public/sites
     restart: always
+    environment:
+      FRANKENPHP_CONFIG: "worker ./public/index.php"
+      SERVER_NAME: :80
+    labels:
+      traefik.enable: true
+      traefik.http.routers.frankenphp-drupal.entrypoints: https
+      traefik.http.routers.frankenphp-drupal.rule: Host(`frankenphp-drupal.docker.so`)
+      traefik.http.routers.frankenphp-drupal.tls: true
+      traefik.http.services.frankenphp-drupal.loadbalancer.server.port: 80
+      traefik.docker.network: stonehenge-network
+    networks:
+      - default
+      - stonehenge-network
 
   postgres:
-    image: postgres
+    container_name: db
+    build:
+      context: .
+      dockerfile: db.Dockerfile
+    image: frankenphp-drupal/postgres
+    volumes:
+      - /var/lib/postgresql/data
     environment:
       POSTGRES_PASSWORD: example
     restart: always
```

Comment:
Same here, can we remove this?

Reply:
I'll move them to another branch on our fork.